doc/doc-site/docs/off-chain/dss/changelog.md

```diff
@@ -13,11 +13,18 @@ Stable DSS builds are tracked on the [`dss-stable`] branch with tags like [`dss-
 1. Merge `main` into `dss-stable`.
 1. Push annotated tag to head of `dss-stable`.
 
-## [v2.1.0] (in progress)
+## [v2.1.0-rc1]
 
 ### Added
 
 - Price level events over MQTT (disabled by default, enable by adding `MQTT_PRICE_LEVELS=yes`, see `src/docker/example.env`) ([#753]).
+- Fix the `/tickers` `base_volume_nominal` field ([#749]).
+- Optimize the `/tickers` endpoint ([#731]).
+- Add suspend and resume functionality for the DSS when deployed on GCP ([#736]).
+- Add more details in aggregator logging ([#738]).
+- Fix a bug that would corrupt your balance if your market account handle started with `0x0` ([#732]).
```

Comment:
-> `that would prohibit balance reporting for market account handles starting with ``0x0`` `

Reply:
Also need to log processor PR https://github.com/econia-labs/aptos-indexer-processors/pull/25

Reply:
Another separate processor PR required is

https://github.com/econia-labs/aptos-indexer-processors/pull/24